src/meson.build

```diff
@@ -18,5 +18,5 @@ raylib = library('raylib',
                   source_c,
                   dependencies : [ glfw_dep, gl_dep, openal_dep, m_dep, x11_dep],
                   install : true,
-                  version : '1.8.0')
+                  version : '1.9.2')
```

Comment:
Should both be set to the same new value and if so what's the difference?

Reply:
I removed the changes to `meson.build`, so you can see what changes would be appropriate.

Reply:
It might be best to remove meson.build and src/meson.build completely, since with this PR it doesn't provide anything more than Make/CMake does provide.

Reply:
That would be a possibility as well. I pushed one more commit that does that.